catalog/resource_catalog_test.go

```diff
@@ -50,6 +66,7 @@ func TestCatalogCreateAlsoDeletesDefaultSchema(t *testing.T) {
 		properties = {
 			c = "d"
 		}
+		owner = "administrators"
```

Comment:
could you create new test, where you set the owner? this will give us more confidence that existing behavior is not affected and this PR is only incrementing features, not causing regressions.

Reply:
created new tests